test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -732,6 +787,30 @@ public override void OnStopActivity(Activity activity, object payload)
             }
         }
 
+        private class TestNullHostActivityMiddlewareImpl : ActivityMiddleware.ActivityMiddlewareImpl
+        {
+            private ActivitySource activitySource;
+            private Activity activity;
+            private string activityName;
+
+            public TestNullHostActivityMiddlewareImpl(string activitySourceName, string activityName)
+            {
+                this.activitySource = new ActivitySource(activitySourceName);
+                this.activityName = activityName;
+            }
+
+            public override void PreProcess(HttpContext context)
+            {
+                Activity.Current = null;
```

Comment:
why setting Activity.Current to null here?

Reply:
Was recommended to add this one. 
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3498#discussion_r932650509

Reply:
please describe it in the test case to make it clear what are we testing here.

Reply:
done